engine/spec/components/citizens_advice_components/input_spec.rb

```diff
@@ -112,6 +112,18 @@
     it "renders the hint text" do
       expect(component.text.strip).to include("This is the hint text")
     end
+
+    it "adds aria-describedby to the input" do
+      expect(component.css("input").attribute("aria-describedby").value).to eq("example-input-hint")
+    end
+  end
+
+  context "when there is hint text and an error" do
+    let(:options) { { hint: "this is the hint text", error_message: "Enter your name" } }
+
+    it "adds aria-describedby to the input" do
+      expect(component.css("input").attribute("aria-describedby").value).to eq("example-input-error example-input-hint")
+    end
   end
```

Comment:
This is great. Could we add one more test to check that there's no `aria-describedby` when there's no error or hint? I reckon right now it'll output an empty attribute which is probably not what we want. Or maybe Rails is smart enough to work that out?

Reply:
Good call, I'll add one.